### PR TITLE
refactor: use new design system header

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-545

### CODE REVIEW

**Review by luizstacio — CHANGES_REQUESTED** [review]
Can you implement the mobile one also?

**Comment by LuizAsFight** [comment]
in the Nav component we can also include the `network` , `account` and `onConnect` props to make it like this one ?
https://design.fuel.network/?path=/story/ui-nav--usage

@luizstacio ok ?

**Comment by luizstacio** [comment]
> in the Nav component we can also include the `network` , `account` and `onConnect` props to make it like this one ? https://design.fuel.network/?path=/story/ui-nav--usage
> 
> @luizstacio ok ?

We can open an issue to prioritize this in the next cycle.

File: packages/app/src/systems/Core/components/Header.tsx [hunk 1]
```diff
@@ -1,126 +1,56 @@
...
+          >
+            Bridge
+          </Nav.MenuItem>
+          <Nav.MenuItem
+            href={Pages.ecosystem}
+            isActive={isLinkActive(Pages.ecosystem)}
+          >
+            Ecosystem
+          </Nav.MenuItem>
+        </Nav.Menu>
```
Comment (luizstacio) [inline packages/app/src/systems/Core/components/Header.tsx:31]: We need to keep the logic that uses the `IS_PREVIEW`; They ensure that when we are on the ecosystem page, the menu items are not shown.

**Review by luizstacio — CHANGES_REQUESTED** [review]

**Review by luizstacio — CHANGES_REQUESTED** [review]
Can you take a look why page refreshes on each click, it should just change the route using react router.

**Review by luizstacio — APPROVED** [review]

**Pull request merged**